lib/base_palettes.hpp

```diff
@@ -147,7 +147,7 @@ class Palette
 // Since there is no make_unique for c++11
 #if __cplusplus < 201402
```

Comment:
And when it is higher, `as_unique` will not be defined 😅

Reply:
oops haha, thank you